testing/tcphandler/main.go

```diff
@@ -1,15 +1,21 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net"
+
+	"github.com/sensu/sensu-go/types"
 )
 
 var (
 	port = flag.Int("port", 3333, "port to run tcp server on")
 	conn = flag.String("type", "tcp", "type of connection to establish (ex. tcp)")
+	file = flag.String("file", "/tmp/tcp-handler.json", "optionally write json to file (ex. /tmp/tcp-handler.json")
```

Comment:
I'd call that flag `-out` or `-o` to clearly indicate that the given argument is purely to output things into (and be in line with most other go tools I think?)